src/features/apiDelegation/offered/ChooseApiPage/ChooseApiPage.tsx

```diff
@@ -212,12 +208,16 @@ export const ChooseApiPage = () => {
             <div className={classes.pageContentActionBarsContainer}>
               <div className={common.apiAccordions}>
                 <h4 className={classes.explanationTexts}>{t('api_delegation.delegable_apis')}:</h4>
-                <div className={classes.delegableApisContainer}>{delegableApiAccordions()}</div>
+                <div className={classes.delegableApisContainer}>
+                  <div className={classes.actionBarWrapper}>{delegableApiActionBars()}</div>
+                </div>
               </div>
               {!isSm && (
                 <div className={common.apiAccordions}>
                   <h4 className={classes.explanationTexts}>{t('api_delegation.chosen_apis')}</h4>
-                  <div className={classes.delegableApisContainer}>{chosenApiAccordions}</div>
+                  <div className={classes.delegableApisContainer}>
+                    <div className={classes.actionBarWrapper}>{chosenApiAccordions}</div>
```

Comment:
Should probably rename this one too
```suggestion
                    <div className={classes.actionBarWrapper}>{chosenApiActionBars}</div>
```